src/TinyHelpers.AspNetCore/Attributes/ContentTypeAttribute.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+using Microsoft.AspNetCore.Http;
+
+namespace TinyHelpers.AspNetCore.Attributes;
+
+[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter, AllowMultiple = false)]
+public class ContentTypeAttribute : ValidationAttribute
+{
+    private readonly string[] validContentType;
+    private readonly string[] imageContentTypes = new string[] { "image/jpeg", "image/png", "image/gif" };
+    private readonly string[] videoContentTypes = new string[] { "video/mp4", "video/ogg", "video/quicktime" };
+
+    public ContentTypeAttribute(string[] ValidContentType)
+    {
+        validContentType = ValidContentType;
```

Comment:
Poiché è l'utente a passare esplicitamente la lista, è necessario applicare una normalizzazione dei valori (ad esempio, assicurarsi che tutti i valori siano in minuscolo).